util/stmtsummary/statement_summary.go

```diff
@@ -1022,3 +1041,19 @@ func convertEmptyToNil(str string) interface{} {
 	}
 	return str
 }
+
+func newInduceSsbd(beginTime int64, endTime int64) *stmtSummaryByDigest {
+	newSsbd := &stmtSummaryByDigest{
+		history: list.New(),
+	}
+	newSsbd.history.PushBack(newInduceSsbde(beginTime, endTime))
+	return newSsbd
+}
+func newInduceSsbde(beginTime int64, endTime int64) *stmtSummaryByDigestElement {
+	newSsbde := &stmtSummaryByDigestElement{
+		beginTime:  beginTime,
+		endTime:    endTime,
+		minLatency: time.Duration.Round(1<<63-1, time.Nanosecond),
+	}
+	return newSsbde
+}
```

Comment:
Look like this function only use in test, prefer to move this into `evicted_test.go`